micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Changelog 
 
+## [1.8.13]  [16-Dec-2024]
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)